pkg/controller/controllerutil/controllerutil.go

```diff
@@ -91,6 +91,29 @@ func SetControllerReference(owner, controlled metav1.Object, scheme *runtime.Sch
 	return nil
 }
 
+// RemoveControllerReference is a helper method to make sure the given object removes an controller reference to the object provided.
+// This allows you to remove the owner to establish a new owner of the object in a subsequent call.
+func RemoveControllerReference(owner, controlled metav1.Object) error {
```

Comment:
```suggestion
func RemoveControllerReference(owner, object metav1.Object) error {
```

Reply:
Resolving as I updated this on the latest push.